.gitignore

```diff
@@ -1,3 +1,7 @@
+.classpath
+.project
+.settings/
+package-lock.json
```

Comment:
@zromick Should package-lock.json be a part of .gitignore? 
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
Removed as discussed!